pkg/entities/nfts.go

```diff
@@ -429,25 +429,33 @@ func (e *Entity) CreateEVMNFTCollection(req request.CreateNFTCollectionRequest)
 		e.log.Errorf(err, "[util.ConvertChainToChainId] failed to convert chain to chainId: %v", err)
 		return nil, fmt.Errorf("Failed to convert chain to chainId: %v", err)
 	}
-
+	colNolFound := false
 	image, err := e.getImageFromMarketPlace(chainID, req.Address)
 	if err != nil {
 		e.log.Errorf(err, "[e.getImageFromMarketPlace] failed to get image from market place: %v", err)
-		return nil, err
+		colNolFound = strings.Contains(err.Error(), "Collection not found") && strings.Contains(err.Error(), "paintswap")
+		// if the error not related to collection not found should return
+		if !colNolFound {
+			return nil, err
```

Comment:
check lại ở cả phía bot liệu return err có được ko. Khi user add collection ko có trên paintswap (trên nftkey, ...) thì vẫn add được bth

Reply:
check thì khi collection not found sẽ chuyển tới lấy name và symbol, ở local indexer ko có migration nên ko add được dưới local

Reply:
m comment khúc gọi qua indexer đi, r test lại. Make sure là vẫn add được nhé

Reply:
<img width="420" alt="Screenshot 2022-11-15 at 16 59 06" src="https://user-images.githubusercontent.com/49946656/201889817-60de23c3-8d5f-4ff1-9118-1b3283707316.png">
<img width="605" alt="Screenshot 2022-11-15 at 16 59 17" src="https://user-images.githubusercontent.com/49946656/201889877-2240fb9b-5247-4c4a-a831-a47ee160f6f2.png">
Bỏ qua gọi indexer add được nha m